internal/schema/database.go

```diff
@@ -41,6 +42,7 @@ func (d *Database) ToProto() proto.Message {
 		Name:     d.Name,
 		Type:     d.Type,
 	}
+	// We are not serializing runtime, as it can contain sensitive information
```

Comment:
If it's never serialised, how does it ever hit the deserialise code path?

Reply:
I think it would probably be preferable to serialise here, but add a "Redact()" visitor function to the schema that redacts sensitive fields, which we would call before serialising the schema to the database.